protocol-designer/src/pages/Designer/DeckSetup/utils.ts

```diff
@@ -160,21 +160,48 @@ export const getLabwareIsRecommended = (
 //  purely for labware<>adapter combos
 export const getLabwareCompatibleWithAdapter = (
   defs: LabwareDefByDefURI,
+  enableStackingFF: boolean,
   adapterLoadName?: string
 ): string[] => {
   if (adapterLoadName == null) {
     return []
   }
-  return Object.entries(defs)
-    .filter(
-      ([, { stackingOffsetWithLabware, compatibleParentLabware }]) =>
-        stackingOffsetWithLabware?.[adapterLoadName] != null &&
-        //  stacking labware gets added via the LabwareCard
-        !compatibleParentLabware?.includes(adapterLoadName)
-    )
-    .map(([labwareDefUri]) => labwareDefUri)
+
+  if (enableStackingFF) {
+    return Object.entries(defs)
+      .filter(
+        ([, { stackingOffsetWithLabware }]) =>
+          stackingOffsetWithLabware?.[adapterLoadName] != null
+      )
+      .map(([labwareDefUri]) => labwareDefUri)
+  } else {
+    return Object.entries(defs)
+      .filter(
+        ([, { stackingOffsetWithLabware, compatibleParentLabware }]) =>
+          stackingOffsetWithLabware?.[adapterLoadName] != null &&
+          !compatibleParentLabware?.includes(adapterLoadName)
+      )
+      .map(([labwareDefUri]) => labwareDefUri)
+  }
```

Comment:
I think this would work: 
```suggestion
    return Object.entries(defs).reduce<string[]>(
      (
        acc,
        [labwareDefURI, { stackingOffsetWithLabware, compatibleParentLabware }]
      ) =>
        stackingOffsetWithLabware?.[adapterLoadName] != null &&
        (!enableStackingFF ||
          !compatibleParentLabware?.includes(adapterLoadName))
          ? [...acc, labwareDefURI]
          : acc,
      []
    )
```